### ISSUE TITLE
Bower wiredep fails because of bad path at main.

### ISSUE THREAD

**Issue by scaria** [epochjs/epoch#176]
The grunt wiredep fails because the main has an absolute path.
```
"main": [
    "/epoch.min.js",
    "/epoch.min.css"
  ],
```

Changing this to

```
"main": [
    "epoch.min.js",
    "epoch.min.css"
  ],
```
works.

**Comment by rsandor** [comment]
Seems to be a duplicate of #171

**Comment by rsandor** [comment]
Fixed by #168
